versioned/storage/cache/src/main/java/org/projectnessie/versioned/storage/cache/ObjCache.java

```diff
@@ -55,4 +69,22 @@ public interface ObjCache {
   void putReferenceLocal(@Nonnull Reference r);
 
   void putNegative(@Nonnull String name);
+
+  /**
+   * Special sentinel object instance to indicate that an object has been marked as "not found" via
+   * {@link #putNegative(ObjId, ObjType)}, returned from {@link #get(ObjId)}. This object is only
+   * for cache-internal purposes.
+   */
+  Obj NOT_FOUND_OBJ_SENTINEL =
```

Comment:
Why do we keep this constant here, but `NON_EXISTENT_REFERENCE_SENTINEL` in `CacheBackend`?

Reply:
moved